go/libraries/doltcore/sqle/schema/schema.go

```diff
@@ -33,31 +35,53 @@ import (
 
 var ErrPartiallyDefinedTags = fmt.Errorf("must define tags for all or none of the schema columns")
 
-// doltSchemaToSqlSchema returns the sql.Schema corresponding to the dolt schema given.
-func doltSchemaToSqlSchema(tableName string, sch schema.Schema) (sql.Schema, error) {
-	cols := make([]*sql.Column, sch.GetAllCols().Size())
-
-	var i int
-	err := sch.GetAllCols().Iter(func(tag uint64, col schema.Column) (stop bool, err error) {
-		var innerErr error
-		cols[i], innerErr = doltColToSqlCol(tableName, col)
-		if innerErr != nil {
-			return true, innerErr
+// ApplyDefaults applies the default values to the given indices, returning the resulting row.
+func ApplyDefaults(ctx context.Context, doltSchema schema.Schema, sqlSchema sql.Schema, indicesOfColumns []int, oldRow row.Row) (row.Row, error) {
```

Comment:
s/oldRow/row/

Reply:
So I didn't wanna use `row` because we have a package named `row` and I didn't want that naming conflict